common/components/stores/ProjectSearchStore.js

```diff
@@ -80,17 +83,13 @@ class ProjectSearchStore extends ReduceStore<State> {
     }
   }
 
-  _clearProjects(state: State): State {
-    return state.set('projects', null);
-  }
-
-  _loadProjects(state: State): void {
-    const {keyword, issueArea} = state;
+  _loadProjects(state: State): State {
     const url = [
```

Comment:
Props for being able to cram all the url building logic into a single expression, but I'm wondering if the more boring pattern:
if(state.x) {
  url += '&x=' + yaddayadda;}
might be more readable and maintainable in the long run.

Reply:
ty for the props <3 :3

I've refactored some in d60801a, lemme know whatcha think

Reply:
Looks good!